tests/x11/thunderbird.pm

```diff
@@ -18,11 +17,15 @@ use testapi;
 
 sub run {
     ensure_installed("MozillaThunderbird");
-    x11_start_program('thunderbird');
-    wait_screen_change {
-        send_key "alt-f4";    # close wizard
-    };
-    send_key "alt-f4";        # close prog
+
+    x11_start_program("thunderbird", target_match => "test-thunderbird-1");
+    assert_screen "test-thunderbird-1";
```

Comment:
That does not seem to make sense because `x11_start_program` should already conduct the screen matching. Also, please just use the existing needle tag so that you do not need to specify the `target_match` parameter even. I do not see added value in the name "test-thunderbird-1".

Reply:
I remember that I had an issue, but I tried so many stuff back then that I have to double check this one.